.github/workflows/.test.yml

```diff
@@ -28,7 +28,7 @@ env:
   DOCKER_EXPERIMENTAL: 1
   DOCKER_GRAPHDRIVER: ${{ inputs.storage == 'snapshotter' && 'overlayfs' || 'overlay2' }}
   TEST_INTEGRATION_USE_SNAPSHOTTER: ${{ inputs.storage == 'snapshotter' && '1' || '' }}
-  SETUP_BUILDX_VERSION: latest
+  SETUP_BUILDX_VERSION: v0.18.0
```

Comment:
Not for this PR, but now wondering if we should set a global version, and also override the one we use in the Dockerfile? https://github.com/moby/moby/blob/f052dbb54bdfd4dfcb8b007c0dbf33f533ecffd7/Dockerfile#L15

Although that would mean that we would not get early warnings on failures in rc's; is there a value other than `latest` that also includes betas and rc's?

Reply:
Yes having some kind of `latest-unstable` would be good to catch this early